adapters/Nend/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ## nend iOS Mediation Adapter Changelog
 
+#### Version 5.4.0.0
+- Verified compatibility with nend SDK 5.4.0
+- Added support for native ad.
+
+Built and tested with
+- Google Mobile Ads SDK version 7.53.1.
```

Comment:
Please use the latest SDK version `7.56.0` ?

Reply:
OK. I'll update version to `7.56.0`.